src/cassandra.go

```diff
@@ -27,7 +28,7 @@ type argumentList struct {
 
 const (
 	integrationName    = "com.newrelic.cassandra"
-	integrationVersion = "2.4.0"
+	integrationVersion = "2.4.1"
```

Comment:
We should automate this as it catches so many people out. Guess we should add it to the backlog.

Reply:
Added this in the backlog long time ago :D 
https://newrelic.atlassian.net/browse/IHOST-2925